resource/eastwood/config/clojure.clj

```diff
@@ -32,3 +32,8 @@
   :within-depth 3
   :reason "when-some with an empty body is warned about, so warning about let with an empty body in its macroexpansion is redundant."})
 
+(disable-warning
+ {:linter :constant-test
+  :if-inside-macroexpansion-of #{'clojure.core/cond-> 'clojure.core/cond->>}
```

Comment:
Can we also add `'clojure.core/cond` here too?

Reply:
Is that necessary? From [here](https://github.com/jonase/eastwood/blob/eb58d6fb2628e1d178d4b8d2c46d4788b2a20094/cases/testcases/constanttestexpr.clj#L46) it looks to me like that is handled already.